src/jdk.compiler/share/classes/com/sun/tools/javac/code/Symbol.java

```diff
@@ -667,7 +667,7 @@ public Symbol asMemberOf(Type site, Types types) {
      *  It is assumed that both symbols have the same name.  The static
      *  modifier is ignored for this test.
      *
-     *  See JLS 8.4.6.1 (without transitivity) and 8.4.6.4
+     *  See JLS 8.4.8.1 (without transitivity) and 8.4.8.4
```

Comment:
Any idea what the "(without transitivity)" is referring to here and elsewhere?

Reply:
My guess is that "transitivity" here refers to the subclass relationship being the transitive closure of the direct subclass relationship. Could it also be that the "quirk" paragraph starting at com/sun/tools/javac/code/Symbol.java:2057 is relevant here? @mcimadamore?

Reply:
FWIW, I found a related bug: https://bugs.openjdk.java.net/browse/JDK-4362349. It might be responsible for that "(without transitivity)" caveat.